cmd/crc/cmd/config/config.go

```diff
@@ -36,8 +36,9 @@ func less(lhsKey, rhsKey string) bool {
 	return lhsKey < rhsKey
 }
 
-func configurableFields(config *config.Config) string {
+func configurableFields(config, secretConfig *config.Config) string {
```

Comment:
with https://github.com/code-ready/crc/pull/3333/commits/fe25223cb8d7369954bffc4d34c2d88e5c4a863a#r969320847 the goal is to remove having to pass in two config structs